README.md

```diff
@@ -1,14 +1,61 @@
-# XAITK - Saliency
+![xaitk-logo](./docs/figures/xaitk-wordmark.png)
 
-## Intent
-Provide interfaces that convey a standard API for visual saliency
-map generation.
+<hr/>
+
+[![Documentation Status](https://readthedocs.org/projects/xaitk-saliency/badge/?version=latest)](https://xaitk-saliency.readthedocs.io/en/latest/?badge=latest)
+[![badge-unittests](https://github.com/xaitk/xaitk-saliency/actions/workflows/ci-unittests.yml/badge.svg)](https://github.com/XAITK/xaitk-saliency/actions/workflows/ci-unittests.yml)
+[![badge-notebooks](https://github.com/xaitk/xaitk-saliency/actions/workflows/ci-example-notebooks.yml/badge.svg)](https://github.com/XAITK/xaitk-saliency/actions/workflows/ci-example-notebooks.yml)
+[![codecov](https://codecov.io/gh/XAITK/xaitk-saliency/branch/master/graph/badge.svg?token=VHRNXYCNCG)](https://codecov.io/gh/XAITK/xaitk-saliency)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=XAITK_xaitk-saliency&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=XAITK_xaitk-saliency)
+[![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/XAITK/xaitk-saliency.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/XAITK/xaitk-saliency/context:python)
+
+The `xaitk-saliency` package is an open-source, explainable AI framework for
+visual saliency algorithm interfaces and implementations, built for
+analytics and autonomy applications.
+
+See [here](https://xaitk-saliency.readthedocs.io/en/latest/introduction.html)
+for a more formal introduction to the topic of XAI and visual saliency
+explanations.
+
+This framework is a part of the [Explainable AI Toolkit (XAITK)](
+https://xaitk.org).
+
+## Target Audience
+This toolkit is intended to help data scientists and developers who want to
+add visual saliency explanations to their workflow or product for relevant
+models.
+Functionality provided here is both directly accessible for targeted
+experimentation, and through Strategy and Adapter patterns to allow for
+modular integration into systems and applications.
+
+## Installation
+Install the latest release via pip:
+```bash
+pip install xaitk-saliency
+```
+
+Some plugins may require additional dependencies in order to be utilized at
+runtime.
+Such details are described [here](
+https://xaitk-saliency.readthedocs.io/en/latest/implementations.html).
+
+See [here for more installation documentation](
+https://xaitk-saliency.readthedocs.io/en/latest/installation.html).
+
+## Getting Started
+We provide a number of Jupyter-notebook based examples in the `./examples/`
+directory to show example usage of the `xaitk-saliency` package in a number of
```

Comment:
We use "example" a lot in this sentence here, I would recommend dropping the last one:

```suggestion
directory to show usage of the `xaitk-saliency` package in a number of
```